code/modules/research/experimentor.dm

```diff
@@ -543,7 +544,7 @@
 		usr << browse(null, "window=experimentor")
 		return
 	else if(scantype == "search")
-		var/obj/machinery/computer/rdconsole/D = locate(/obj/machinery/computer/rdconsole) in oview(3,src)
+		var/obj/machinery/computer/rdconsole/D = locate(/obj/machinery/computer/rdconsole) in oview(range_search,src)
```

Comment:
Same reasoning as with Kangaroos here. Added a variable to allow for easier in-map tweaking of consoles.